chia/util/db_wrapper.py

```diff
@@ -1,7 +1,13 @@
+from __future__ import annotations
+
 import asyncio
+import contextlib
+from typing import AsyncIterator, Dict, Optional
 
 import aiosqlite
 
+savepoint_name: int = 0
```

Comment:
it just occurred to me that this might as well be a member of the `DBWrapper2` class